gitlab/datadog_checks/gitlab/gitlab_v2.py

```diff
@@ -51,13 +51,12 @@ def check(self, _):
             super().check(_)
         finally:
             # Service check to check GitLab's health endpoints
-            if self.config.gitlab_url is not None:
+            if self.config.gitlab_url:
```

Comment:
This was also failing if it was an empty string, which is what is happening on the unit test when no value is provided